packages/xchain-thorchain/src/types/proto/MsgDeposit.d.ts

```diff
@@ -0,0 +1,626 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Where is the original `proto` file for `MsgDeposit` defined? If needed, pls add a description in the README how to use `*.proto` files to generate `MsgDeposit.{d.ts|js}` files

Reply:
done